CSC322-Grinnell/mayflower-directory
# Fix Directory Results Formatting

Resolves #167 , #154 

# Changes

- fix link on reset button click in `directory.html.erb`(#167)
- format reset button to look like search button, `directory.html.erb` and `custom.scss`
- fix formatting when searching for residents from a specific neighborhood over 98 results in `directory.html.erb` (#154)

# Testing

0. Run `db rake:dummy_seed` if your local database is not already populated. The command can be used in succession to create distinct profiles in the database.  
1. Login and navigate to the Directory page,
2. Search by neighborhood,
4. Click reset button->search results should reset. 

**Require >98 search results to confirm the problem is resolved, but search results should now display correctly regardless of number of results**
